packages/nx-plugin/src/utils/testing-utils/commands.ts

```diff
@@ -35,11 +37,17 @@ export function runNxCommand(
   }
 }
 
-export function runCommand(command: string): string {
+export function runCommand(
+  command: string,
+  opts: { env?: NodeJS.ProcessEnv } = {
+    env: {},
+  }
```

Comment:
I think this could be optional. 
```suggestion
  opts?: { env?: NodeJS.ProcessEnv }
```

Reply:
agree, that default value does not add... value